tests/space-time-stack/test_demangling.cpp

```diff
@@ -0,0 +1,90 @@
+#include <iostream>
+#include <regex>
+#include <sstream>
+
+#include "Kokkos_Core.hpp"
+#include "kp_all.hpp"
+
+#include "utils/demangle.hpp"
+
+template <typename execution_space, size_t size = 5>
```

Comment:
Why would you want to template the whole struct?
Get rid of `apply()`, move the test to the constructor that you template on `ExecutionSpace`.  Also there is no point on making the number of iteration a template parameter.

Reply:
* I think I templated the *struct* over the *execution_space* type to check what the tool would output when looking at kernels defined in templated classes. If you think it is unnecessary, I can get rid of the *execution_space* template.
* For the number of iterations, I think I wanted to have a compile-time range, but I guess it is useless for this test. I'll use a range from 0 to 1 now :wink: